### PR TITLE
Feature/update chain integration & add swagger

### PR DESCRIPTION
**Pull request by kether-c**
## Update chain-integration.md
1. To have both Cronos mainnet and testnet integration documentation
2. Add JSON-RPC swagger playground to both mainnet and testnet doc
3. Remove curl JSON-RPC samples

### CODE REVIEW

File: docs/resources/chain-integration.md [hunk 1]
```diff
@@ -5,23 +5,19 @@
...
-- [Binaries](https://github.com/crypto-org-chain/cronos/releases)
-
-- [Seeds for fullnode](https://github.com/crypto-org-chain/cronos-testnets/blob/main/testnet.json#L21)
-- Genesis files
-  - [Mainnet genesis](https://raw.githubusercontent.com/crypto-org-chain/cronos-mainnet/master/cronosmainnet_25-1/genesis.json)
-  - [Testnet genesis](https://github.com/crypto-org-chain/cronos-testnets/blob/main/cronostestnet_338-1/genesis.json)
+- [Cronos Binaries](https://github.com/crypto-org-chain/cronos/releases)
 
-## Setup Guide
+## Integration guide for **Cronos Mainnet**
```
Comment (lezzokafka) [inline docs/resources/chain-integration.md:10]:
```suggestion
## Integration guide for **Cronos Mainnet Beta**
```

File: docs/resources/chain-integration.md [hunk 2]
```diff
@@ -60,12 +56,21 @@ To provide a stable experience to users, there is a request rate limit on the pu
...
 
-- [**TypeScript** library](https://github.com/crypto-org-chain/chain-jslib);
-- [**Python** library](https://pypi.org/project/chainlibpy/#description);
-- [**Rust** library](https://github.com/crypto-org-chain/chainlib-rs) (note that it is not feature complete);
-- [@cosmjs/stargate](https://github.com/cosmos/cosmjs/tree/master/packages/stargate).
+- [Joining the Cronos Testnet](https://cronos.crypto.org/docs/getting-started/cronos-testnet.html)
+- [Testnet Faucet](https://cronos.crypto.org/faucet/)
+  - Note that the faucet only takes eth-type `0x...` addr
+- [Seeds for fullnode](https://github.com/crypto-org-chain/cronos-testnets/blob/main/testnet.json#L21)
+- [Genesis files](https://github.com/crypto-org-chain/cronos-testnets/blob/main/cronostestnet_338-1/genesis.json)
```
Comment (lezzokafka) [inline docs/resources/chain-integration.md:66]: Very minor suggestion: Just to align with the mainnet one and easier for download/`cp`. 
```suggestion
- [Genesis files](https://raw.githubusercontent.com/crypto-org-chain/cronos-testnets/main/cronostestnet_338-1/genesis.json)
```

File: docs/resources/chain-integration.md [hunk 3]
```diff
@@ -42,7 +38,7 @@ To provide a stable experience to users, there is a request rate limit on the pu
 4. **EVM HTTP JSON RPC (Web3 compatible)**
    - https://evm-cronos.crypto.org
 
-## Official token contract addresses for Cronos mainnet Beta
+### Official token contract addresses for Cronos mainnet Beta
```
Comment (lezzokafka) [inline docs/resources/chain-integration.md:41]: @kether-c Kindly update the link to https://cronoscan.com/ in the cases for Cronos mainnet Beta

For example:
```
https://cronos.crypto.org/explorer/address/0x02DCcaf514C98451320a9365C5b46C61d3246ff3/transactions
```
->
```
https://cronoscan.com/address/0x02DCcaf514C98451320a9365C5b46C61d3246ff3
```

**Review by lezzokafka — COMMENTED** [review]
Left some comments and suggestions. Thanks for deploying the swagger playground 👍

File: docs/resources/chain-integration.md (hunk 3 above)
Comment (kether-c) [reply docs/resources/chain-integration.md:41]: Sure, updated and pushed

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**